modules/javascript/partytown-web-worker/load.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/**
+ * Module Name: Partytown Web Worker
+ * Description: Run JavaScript in a separate Web Worker with the help of Partytown library.
+ * Experimental: Yes
+ *
+ * @since   1.0.0
+ * @package performance-lab
+ */
+
+/**
+ * PartyTown Configuration
+ *
+ * @since 1.0.0
+ * @see https://partytown.builder.io/configuration
+ * @return void
+ */
+function perflab_web_worker_partytown_configuration() {
+	$config = array(
+		'lib'     => str_replace( site_url(), '', plugin_dir_url( __FILE__ ) ) . 'assets/js/partytown/',
```

Comment:
Does partytown require https? if so we should check that the site url does as well

Reply:
Yes, it would, since service workers require HTTPS. That is, unless the site is `localhost` in which browsers will it like HTTPS. 

Partytown does not require service workers, but they can be easier to deploy. The better approach is using [Atomics](https://partytown.builder.io/atomics), but it has other requirements with COOP/COEP.

Reply:
Humm, that is a good case to consider. In that case, it would be better to disable the module itself if the site is not using SSL with a notice like:
```txt
The partytown web worker module only works if the site is SSL enabled.
```

Reply:
Yes, that makes sense to me, if not `localhost`.

Reply:
Added in https://github.com/WordPress/performance/pull/556/commits/4191312440cca374e0c377f8b277acacc887ab5e